generated/niscope/niscope/_library.py

```diff
@@ -8,6 +8,8 @@
 
 import niscope.waveform_info as waveform_info  # noqa: F401
 
+import niscope.measurement_stats as measurement_stats  # noqa: F401
```

Comment:
[Codegen templates.](https://github.com/shilohbeeler-NI/nimi-python/blob/02859f9d093e4251dae907f8e023d31f4887811e/build/templates/_library.py.mako#L21)

Reply:
That codegen template is intended for custom types defined in C driver library. We had considered adding some logic to avoid generating for Python-only custom types but decided that added metadata complexity to differentiate isn't worth it.